stk/ops/linear_ops_test.py

```diff
@@ -192,121 +193,10 @@ def testLinearOps_Sdd(self, m, k, n, trans_a, trans_b, blocking, sparsity):
         self.assertEqual(expected_grad.size()[1], grad.size()[1])
         self.assertTrue(allclose(grad, expected_grad))
 
-    def testLinearOps_Ssd(self, m, k, n, trans_a, trans_b, blocking, sparsity):
-        # Construct the operands.
-        a_shape = (k, m) if trans_a else (m, k)
-        a_dense, a = _dense_and_sparse(*a_shape, sparsity, blocking)
-        b_shape = (n, k) if trans_b else (k, n)
-        b, bcp = _dense_2x(*b_shape)
+    def testRowIndices(self, m, k, n, trans_a, trans_b, blocking, sparsity):
```

Comment:
Do we need this test? I think it's comparing ops.row_indices to a reference result that was also created with ops.row_indices...